bigtable/bigtable.go

```diff
@@ -119,6 +119,7 @@ func NewClient(ctx context.Context, project, instance string, opts ...option.Cli
 
 // NewClientWithConfig creates a new client with the given config.
 func NewClientWithConfig(ctx context.Context, project, instance string, config ClientConfig, opts ...option.ClientOption) (*Client, error) {
+	elaspedTimer := time.Now()
```

Comment:
typo here. Also, this is a bit confusing that this is called a timer while this is just a point in time -- same applies to `startupTimer` argument of `NewBigtableChannelPool`

Reply:
Fixed.